### PR TITLE
Feature/128  Fit to Window (2nd approach)

### LINKED ISSUE #128: Introduce Fit-To-Window control to mapping toolbar
As a < Player >, I want < to reset the zoom > so that < I can quickly see the whole map >.

# Acceptance criteria

- [ ] New button present on mapping toolbar
- [ ] Clicking the button zooms out to show full extent of map

# UX
See the "fit to window" icon in the toolbar here:
image

# Notes
It's fine to pull in another open source library, if necessary.  It _probably_ isn't necessary just for this feature.

### PR DESCRIPTION
**Pull request by IanMayo**
## 🧰 Ticket
Fixes #128

## 🚀 Overview: 
Add existing control that introduces fit-to-window functionality

## 🔗 Link to preview
https://serge-review-leaflet-ho-h2xdub.herokuapp.com/

## 🖥️ Screenshot
[If the work is UI related then paste a screenshot of the update here.]
image

### CODE REVIEW

File: packages/client/src/Components/Mapping/index.jsx [hunk 1]
```diff
@@ -8,6 +9,7 @@ import markerFor from '../../Helpers/markerFor'
 import '../../Helpers/mousePosition'
 
 import './styles.scss'
+import './leaflet.zoomhome.js'
```
Comment (foxleigh81) [inline packages/client/src/Components/Mapping/index.jsx:14]: Can you add a `//TODO: Refactor` above this line. It's fine for now but I don't want us to get into the habit of doing naked imports on javascript. It's not worth doing for the Feb demo as it requires converting `zoomhome` into an actual model but it should definitely be done for the rebuild.

**Review by foxleigh81 — DISMISSED** [review]
Approved but if you could add the comment I mentioned, that'd be super.

**Comment by IanMayo** [comment]
Closing, submitted in #151, where the changes have been applied to the current mapping control

**Comment by IanMayo** [comment]
Closing, submitted in #151, where the changes have been applied to the current mapping control

**Pull request closed without merging**